src/chocolatey.resources/helpers/functions/Get-ChocolateyWebFile.ps1

```diff
@@ -261,8 +261,18 @@ param(
 
   # If we're on 32 bit or attempting to force 32 bit and there is no
   # 32 bit url, we need to throw an error.
-  if ($url -eq $null -or $url -eq '') {
-    throw "This package does not support $bitPackage architecture."
+  if (Get-OSArchitectureWidth 32) {
+      if ($url -eq $null -or $url -eq '') {
+        throw "$url was not given."
+  }
+  }
+
+  # If we're on 64 bit or attempting to force 64 bit and there is no

```

Comment:
As far as I'm aware, there is no way to force 64 bit?

Reply:
I haven't found any way to force 64bit